src/common/SmartUrlInput.js

```diff
@@ -22,7 +22,7 @@ type Props = {
   shortAppend: string,
   style?: StyleObj,
   onChange: (value: string) => void,
-  onSubmitEditing: () => Promise<void>,
+  onSubmitEditing: () => void,
```

Comment:
Why this?

Reply:
I did this because the tests were failing, since the onSubmitEditing prop expected an async function (tryRealm), which returned a Promise.